server_v2.py

```diff
@@ -12,6 +12,9 @@
 import c3
 
 logger = logging.getLogger("server")
+http_persistent_db = os.getenv("HTTP_PERSISTENT_DB")
+ws_persistent_db = os.getenv("WS_PERSISTENT_DB")
```

Comment:
Do you need to share databases between different WebSocket connections? If so, then maybe it would be better to just use a single database file for all streams on the server (whether HTTP or WebSockets), and update existing WebSocket tests to not assume that they run on a clean database. I would like to keep the Hrana server simple by not accepting any configuration :)

Reply:
Yeah, the more I think about it, the more sense it makes: let's just use a single temporary database file for the whole server.

Reply:
We cannot use `:memory:`, because we can have only a single connection to an in-memory database, but both Hrana and HTTP need to be able to open multiple connections.

To prevent the locking errors, the server does its own locking (that's what `app["http_db_lock"]` is about). To share the same database between HTTP and WebSockets, this lock must also be acquired by the WebSocket code.

We can merge this as-is and I will change it later.

Reply:
ok, but using `:memory:` seems to work nice here